proxy/operator_resource/resource_manager.py

```diff
@@ -121,6 +123,24 @@ def free_resource(self, is_good_resource: bool, op_resource: OpResourceModel) ->
                 _LOG.debug("disable resource: %s", op_resource)
                 op_signer.disabled_holder_list.append(op_holder)
 
+            try:
+                sol_balance = await self._sol_client.get_balance(op_resource.owner)
+                self._stat_client.commit_op_resource_sol_balance(
+                    OpResourceSolBalanceData(owner=op_resource.owner, balance=sol_balance)
+                )
+
+                neon_address = NeonAccount.from_raw(op_resource.eth_address, op_resource.chain_id)
+                neon_account = await self._core_api_client.get_neon_account(neon_address)
```

Comment:
You should add the operator balance too:
`await self._core_api_client.get_operator_account(...)`

Reply:
Oh!

So have the normal account  AND the operator account. Ok no problem.